server/src/api/modules/groups/services/EditGroupService.ts

```diff
@@ -175,19 +182,47 @@ async function executeUpdate(params: EditGroupParams, updatedGroupFields: Prisma
   const joinedEvents: MemberJoinedEvent[] = [];
   const changedRoleEvents: MemberRoleChangeEvent[] = [];
 
+  const nameChanges = await prisma.nameChange.findMany({
+    where: {
+      OR: missingPlayers.map(p => {
+        const equals = p.username;
+        return { newName: { equals, mode: 'insensitive' } };
+      }),
+      status: NameChangeStatus.PENDING
+    },
+    orderBy: { createdAt: 'desc' }
+  });
+
   await prisma
     .$transaction(async transaction => {
-      // Remove any players that are no longer members
-      const removedPlayerIds = await removeExcessMemberships(
+      const excessMemberships = await removeExcessMemberships(
         transaction,
         params.id,
         memberships,
         nextUsernames
       );
 
+      const removeFromAdd: number[] = [];
```

Comment:
I'd rename these to something like `ignoreFromJoined` and `ignoreFromLeft` because that's technically what we're doing, ignoring them (not removing)